### DISCUSSION TITLE
POST https://aws.connect.psdb.cloud/psdb.v1alpha1.Database/Execute 401

### DISCUSSION THREAD

**Discussion by Visama396** [planetscale/discussion#527] [Bug Reports] [answered]
I'm trying to retrieve data from my table. when using DataGrip it works fine, but when I use the javascript driver in Next.js 13 I get the following error "POST https://aws.connect.psdb.cloud/psdb.v1alpha1.Database/Execute 401"
I print the conn variable and that works fine, the error comes on the line
const result = await conn.execute('SELECT * FROM repartos_users')

I don't know why it's happening now. It worked fine yesterday

**Comment by orware** [comment]
Hello @Visama396,

If you are still experiencing this issue, could you please reach out to us over at https://support.planetscale.com/ when you have a moment so we can look into the details for your database?

Please make sure to include the organization and database name in your message and any other details you can share about the way your JavaScript code may be running.

**Comment by mattrobenolt** [comment, marked as answer]
A 401 is "unauthenticated", so it seems your credentials are wrong or have been deleted, etc. I can't tell you which, based on that information tho. :)
